Sming/SmingCore/Network/SmtpClient.cpp

```diff
@@ -170,6 +170,8 @@ void SmtpClient::onReadyToSendData(TcpConnectionEvent sourceEvent)
 						memcpy((token + 1), url.User.c_str(), url.User.length()); // copy user
```

Comment:
Probably it will be simpler if we use initializer. Something like this should work:
```
uint8_t token[tokenLength] = {0}; 
```

Can you make that change and remove lines 173 and 174 in your PR?